.github/workflows/autorelease.yaml

```diff
@@ -0,0 +1,44 @@
+name: autorelease
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+concurrency:
+  group: ci-autorelease-${{ github.repository }}
```

Comment:
one thing I am a bit unsure is that if this runs aside with master both pipelines will try to publish, which could cause conflicts.. is there a way to trigger it after master actions are done?

Reply:
mmmh, I dont understand this. This job only runs once something is merged to master, it cannot run in parallel with any other job, only with itself and that would cancel if a new one was triggered. Can you explain further what you mean?

Also, if this tries to crate a tag that already exists, it just fails, nothing else.

Reply:
I mean, if this runs on master, and then it decides to tag, it will trigger then the release pipeline, which means master and the release pipeline could run at that point in parallel

Reply:
is that a problem now with the tagged repositories? Or is this about pushing the artifacts tot he repo, that they could collide?

Reply:
we could also run this every night and forget about it, that way it may even be better, max of one release per day, in case we merge several things that bump the cos version.

Reply:
changed it to scheduled, also has a workflow dispatch so we  manually trigger if necessary

Reply:
looks good!